R/js_ifelse.R

```diff
@@ -0,0 +1,79 @@
+#' Convert an R logical expression into a JS ternary expression
+#'
+#' @param condition An R logical expression (supports %in% / %notin% / grepl() / comparisons / & |)
+#' @param true_result  String to return when the condition is TRUE. Default is an empty string, which interprets as `null` in JS.
+#' @param false_result String to return when the condition is FALSE. Default is an empty string, which interprets as `null` in JS.
+#' @return A single character string containing a JavaScript ternary expression.
+#' @export
+js_ifelse <- function(condition, true_result = "", false_result = "") {
```

Comment:
Detail:

Can we improve parameter naming? `if_true`, `if_false`? or `if_truthy`, `if_falsy` as suggested by the JS doc: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator